Documentation/CodeSnippets/Extbase/Controllers/BackendControllerInitialize.rst.txt

```diff
@@ -6,7 +6,7 @@
 
    class BackendController extends ActionController
```

Comment:
should we make this final?

Reply:
Not in the scope of this PR. This has to be done in EXT:blog_example before.